docs/cookbook/recipe_custom_view.rst

```diff
@@ -0,0 +1,134 @@
+Creating a Custom Admin View
+============================
+
+Sometimes you may want to create an additional view in your SonataAdmin
+but you do not have or need an Entity to connect it to. You also probably
+want to add it to sidebar with the rest of your admin pages? Here we will
+present a full working example of how is that achievable.
+
+The recipe
+----------
+
+SonataAdmin provides a very straightforward way of adding your own custom
+view.
+
+To do this we need to:
+
+- create an admin class for your custom view
+- extend the ``SonataAdmin:CRUD`` Controller and tell our admin class to
+  use it
+- create a template that will represent your custom view
```

Comment:
Please end bullet points with semi-colon, and the last one with a dot:
- this;
- that;
- and finally, this.